src/qps.cpp

```diff
@@ -989,7 +989,7 @@ void Qps::menu_edit_cmd()
 void Qps::make_command_menu()
 {
     // should clear SIGNAL!!!
-    QAction *act;
+    //QAction *act; - commented out to reduce warnigs when compiled
```

Comment:
I'm afraid this results in a compilation failure.

Please leave it as it was.

Reply:
Ok